geode-docs/managing/cache_snapshots/exporting_a_snapshot.html.md.erb

```diff
@@ -60,15 +60,37 @@ region.getSnapshotService().save(mySnapshot, SnapshotFormat.GEMFIRE);
 Open a gfsh prompt. After connecting to a <%=vars.product_name%> distributed system, at the prompt type:
 
 ``` pre
-gfsh>export data --region=Region --file=filename.gfd 
---member=membername
+gfsh>export data --region=Region --file=FileName.gfd --member=MemberName
 ```
 
-where *Region* corresponds to the name of the region that you want to export, *filename* (must end in .gfd) corresponds to the name of the export file and *membername* corresponds to a member where the region to export is hosted. For example:
+where *Region* corresponds to the name of the region that you want to export, *FileName* (must end in .gfd) corresponds to the name of the export file and *MemberName* corresponds to a member where the region to export is hosted. For example:
```

Comment:
Consider rewording to be 
*MemberName* corresponds to a member that hosts the region.